modules/uhttp/http.go

```diff
@@ -96,9 +96,9 @@ type Config struct {
 type GetHandlersFunc func(service service.Host) []RouteHandler
 
 // New returns a new HTTP module
-func New(hookup GetHandlersFunc, options ...modules.Option) service.ModuleCreateFunc {
+func New(hookup GetHandlersFunc, filters []Filter, options ...modules.Option) service.ModuleCreateFunc {
```

Comment:
hmm. Filters are bound to uhttp, not modules, having it as parameter makes sense here.